python/tests/op_mappers/test_one_hot_op.py

```diff
@@ -0,0 +1,116 @@
+# Copyright (c) 2023 CINN Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import numpy as np
+from op_mapper_test import OpMapperTest, logger
+import paddle
+
+
+class TestOneHotOp(OpMapperTest):
+    def init_input_data(self):
+        self.feed_data = {'x': self.random([1, 32], 'int32', low=0, high=9)}
+        self.depth = 10
+        self.dtype = 5  # Float32
```

Comment:
前面加个`from paddle.fluid.framework import convert_np_dtype_to_dtype_`
```suggestion
        self.dtype = convert_np_dtype_to_dtype_("float32")
```
当然 `convert_np_dtype_to_dtype_`还是移到`set_op_attrs`更好，这里只用设置`self.dtype = "float32"`